README.md

```diff
@@ -103,7 +103,7 @@ retq
 
 ```cpp
 struct interface {
-  virtual ~iworld() noexcept = default;
+  virtual ~interface() noexcept = default;
```

Comment:
There is the same issue with bind interface :/ (it should be ~interface)
Working link -> https://wandbox.org/permlink/4yBTlH1jvdrsRPes

Reply:
I'm not sure how to fix the template param binding: https://wandbox.org/permlink/ja8x7xXpl5OQMH0b

I can get it to work if the type is not used for inheritance. Not sure what's going on.

Reply:
https://wandbox.org/permlink/wqrF7gZGOM7Jusx9